src/Commands/config/ConfigCommands.php

```diff
@@ -175,6 +176,85 @@ public function delete($config_name, $options = []) {
   }
 
   /**
+   * Display status of configuration (differences between stored and active configuration).
+   *
+   * @command config-status
+   * @param $label A config directory label (i.e. a key in \$config_directories array in settings.php).
```

Comment:
What about viewing the changelist as a diff? Thats already offerred by config-import. Can be a follow-up.